pkg/packagekit/updates.jsx

```diff
@@ -245,6 +266,39 @@ function UpdatesList(props) {
     );
 }
 
+function UpdateHistory(props) {
+    if (!props.history)
+        return null;
+
+    var now = Date.now();
+
+    function formatHeading(time) {
+        if (time)
+            return cockpit.format(_("The following packages were updated $0 ago:"), moment.duration(now - time).humanize());
+        return _("The following packages were recently updated:");
+    }
+
+    function formatPkgs(pkgs) {
+        var names = Object.keys(pkgs);
+        names.sort();
+        names.shift(); // remove _time
```

Comment:
Yes, the arrays get initialized with `_time` in `loadHistory()`. I found that simpler than having to do `.indexOf()`, `splice()`, and piece the two halves back together. But I just thought about `filter()` now, replaced with that.